src/model/writer.js

```diff
@@ -867,7 +877,10 @@ export default class Writer {
 	 * The `options.usingOperation` parameter lets you change if the marker should be managed by operations or not. See
 	 * {@link module:engine/model/markercollection~Marker marker class description} to learn about the difference between
 	 * markers managed by operations and not-managed by operations. It is possible to change this option for an existing marker.
-	 * This is useful when a marker have been created earlier and then later, it needs to be added to the document history.
+	 *
+	 * The `options.affectData` parameter allows you to inform the engine, that this marker does not affects the data model.
+	 * It impacts the performance, because changes in markers with set `options.affectData:false` do not cause firing the
```

Comment:
That's quite irrelevant here.

Instead, you could rather explain what does it mean to affect the data. And that it affects how the `change` or `change:data` is fired.